### DISCUSSION TITLE
Did sb allrdy test NotebookLM?

### DISCUSSION THREAD

**Discussion by CobriMediaJulien** [TriliumNext/Trilium#5976] [General]
Hey guys,

I opened this thread to have a look at certain AI technologies which you can use to investigate your notes further. Did sb allready try to use NotebookLM or sth similar?

I thought about trying it out with triliums html exports.

**Comment by Spacemonk1c** [comment]
I would love see a feature like that.. maybe with a "Jan" API support.. i don't what my notes to go in to another server
here is a project which seems interesting 
https://github.com/soulsands/trilium-chat
